mdb.py

```
@@ -656,6 +639,124 @@ def get_all_dbs_and_tables(db, server):
+            else:
+                column_names = []
+                dict_rows = []
+
+            return {
+                'row_count': len(dict_rows),
+                'data': dict_rows,
+                'column_order': column_names,
+            }
+        finally:
+            query_db['cnf']['servers'][server]['conn'].close()
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)